synapse_data_warehouse/synapse_event/tasks/V2.54.1__create_fileupload_event_task.sql

```diff
@@ -0,0 +1,17 @@
+use database {{database_name}}; --noqa: JJ01,PRS,TMP
+
+-- Step 1) Pause the upstream tasks
+alter task synapse_raw.refresh_synapse_warehouse_s3_stage_task suspend;
+alter task synapse_raw.fileupload_task suspend;
+
+-- Step 2) Create the task of interest
+create or replace task synapse_raw.create_fileupload_event_task
+after synapse_raw.fileupload_task
+as
+    create or replace table synapse_event.fileupload_event
+    clone synapse_raw.fileupload;
+
+-- Step 3) Resume the upstream tasks & the task of interest
```

Comment:
Nit: we used to use 
```
-- Resume the ROOT task and its child tasks
SELECT SYSTEM$TASK_DEPENDENTS_ENABLE( 'REFRESH_SYNAPSE_WAREHOUSE_S3_STAGE_TASK' );
```
But this is not something must to be changed.

Reply:
Thanks for reminding me!

Suspending a root task does not suspend all child tasks ([see here](https://github.com/Sage-Bionetworks/snowflake/pull/216#discussion_r2136390740)), but I'll switch to this command since it makes things cleaner.

Reply:
> if we had any tasks unrelated to this ticket which need to stay in a suspended state 

Good point - will keep that in mind for future tasks